main.py

```diff
@@ -200,10 +210,12 @@ def get_summary(txt, lm):
   inventory
 
 seldocs = select_docs(dt, ch, lg, lm, ck, ct)
+summaries = gather_summaries(dt, ch, lg, lm, ck, ct, seldocs)
+st.success('Summaries loaded!')
```

Comment:
Remove this too!

Reply:
I figure this is about the success message, so I will remove